lib/camera.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flame/game.dart';
+import 'package:forge2d/forge2d.dart';
+
+class Forge2DCamera extends Camera {
+  late ViewportTransform viewportTransform;
```

Comment:
Is this correct? This isn't initialised anywhere in this class

Reply:
It is initialized from the game, it should probably be a constructor instead, then it doesn't need to be `late` either.

Reply:
Moved it to a constructor